src/content/docs/index.mdx

```diff
@@ -20,18 +20,24 @@ hero:
 
 import ClassCardGrid from "../../components/ClassCardGrid.astro";
 import TimedRender from "../../components/TimedRender.astro";
-import { Aside } from "@astrojs/starlight/components"
+import { Aside } from "@astrojs/starlight/components";
 
+{ new Date() < new Date("2024-09-06T17:00:00-05:00") &&
 
-{ new Date() < new Date("2024-09-06T17:00:00-05:00") && 
-<Aside type="caution" title="Interested in registering for CS 484 in Fall 2024?">
-If you're interested in registering for this class, you must have turned in the first assignment due on Thursday August 29th. If you did and aren't in the class yet, please follow the instructions mentioned in class and on Piazza.
+<Aside
+  type="caution"
+  title="Interested in registering for CS 484 in Fall 2024?">
+  If you're interested in registering for this class, you must have turned in
+  the first assignment due on Thursday August 29th. If you did and aren't in the
+  class yet, please follow the instructions mentioned in class and on Piazza.
 </Aside>
 }
 
 { new Date() > new Date("2024-12-31T17:00:00-05:00") &&
+
```

Comment:
Please do not introduce whitespace changes into diffs that you're asking others to review - literally everything here is not an actual edit except the last line added to the file.

Reply:
I'll make sure to avoid introducing unnecessary whitespace changes in future commits.